server/controllers/departuresController.test.ts

```diff
@@ -1,7 +1,7 @@
 import type { Request, Response, NextFunction } from 'express'
 import { createMock, DeepMocked } from '@golevelup/ts-jest'
```

Comment:
Just noticed this and another module (fakerjs) are in your normal dependency tree, shouldn't they be in devDependencies? Was just having a nosey at what this module did 👍

Reply:
Good catch! Dunno how they snuck in there